test-functional/network/http/http-stream.test.ts

```diff
@@ -0,0 +1,44 @@
+import http from 'http';
```

Comment:
On other project, we name the jest directory `tests`, I think we should stick to this

Reply:
I know but we have a folder named `test` so naming another folder `tests` only for test done with Jest doesn't seems right

Reply:
Maybe we can name it `jest` ?